src/commands/completion/completion.ts

```diff
@@ -20,13 +23,53 @@ export const completionGenerate = async (options: OptionValues, command: BaseCom
   }
 
   generateAutocompletion(parent)
-
   await install({
     name: parent.name(),
     completer,
   })
 
-  console.log(`Completion for ${parent.name()} successful installed!`)
+  const TABTAB_CONFIG_LINE = '[[ -f ~/.config/tabtab/__tabtab.zsh ]] && . ~/.config/tabtab/__tabtab.zsh || true'
+  const AUTOLOAD_COMPINIT = 'autoload -U compinit; compinit'
+  const zshConfigFilepath = join(process.env.HOME || homedir(), '.zshrc')
+
+  if (
+    fs.existsSync(zshConfigFilepath) &&
+    checkFileForLine(zshConfigFilepath, TABTAB_CONFIG_LINE) &&
+    !checkFileForLine(zshConfigFilepath, AUTOLOAD_COMPINIT)
+  ) {
+    log(`To enable Tabtab autocompletion with zsh, the following line may need to be added to your ~/.zshrc:`)
+    log(chalk.bold.cyan(`\n${AUTOLOAD_COMPINIT}\n`))
+    await inquirer
+      .prompt([
+        {
+          type: 'confirm',
+          name: 'compinitAdded',
+          message: `Would you like to add it?`,
+          default: true,
+        },
+      ])
+      .then((answer) => {
```

Comment:
Generally you wouldn't use `await` and `.then()` you would use one or the other. If you want it to be blocking use `await`, if you want it to not be blocking use `.then()`

```suggestion
    const answer = await inquirer
      .prompt([
        {
          type: 'confirm',
          name: 'compinitAdded',
          message: `Would you like to add it?`,
          default: true,
        },
      ])
```